### DISCUSSION TITLE
Can't upload multiple pictures.

### DISCUSSION THREAD

**Discussion by szabb** [community/community#124843] [GitHub Education]
### Hi everyone,

I'm trying to register using a Hungarian School ID, which has my name on one side of it, and the school's name on the other side, however, the site doesn't let me upload multiple photos, and due to this, i'm always getting rejected. Any idea?

**Comment by davevad93** [comment]
Hi @Szabbiii ,

There is a solution for that, take a picture of the 2 sides and combine them into one image, this way you can upload the picture with the 2 sides of your ID.
